### ISSUE TITLE
Are there any plans to launch Geoprocessor（ArcGIS Engine）in ArcGIS Pro SDK？

### ISSUE THREAD

**Issue by baciZzz** [Esri/arcgis-pro-sdk-community-samples#57]
I've tried to implement a Geoprocessor modeled after Geoprocessor (ArcGIS Engine) in ArcGIS Pro

**Comment by Wolf-K** [comment]
ArcGIS Pro doesn't support ArcGIS Engine type of implementations, instead you might want to consider the 'ArcGIS Maps SDK for .NET':
https://developers.arcgis.com/net/

**Comment by baciZzz** [comment]
> ArcGIS Pro doesn't support ArcGIS Engine type of implementations, instead you might want to consider the 'ArcGIS Maps SDK for .NET': https://developers.arcgis.com/net/

no,i mean ,we need a tool just like [ESRI.ArcGIS.Geoprocessor.Execute(IGPProcess process, ITrackCancel trackCancel)] ,It allows us to call GP more conveniently, without having to pay attention to what parameters are passed and what the order of the parameters is.

When I want to call the GP tool, I just need to pass in a class that specifies the parameters I need to pass in.

**Comment by baciZzz** [comment]
> ArcGIS Pro doesn't support ArcGIS Engine type of implementations, instead you might want to consider the 'ArcGIS Maps SDK for .NET': https://developers.arcgis.com/net/

no,i mean ,we need a tool just like [ESRI.ArcGIS.Geoprocessor.Execute(IGPProcess process, ITrackCancel trackCancel)] ,It allows us to call GP more conveniently, without having to pay attention to what parameters are passed and what the order of the parameters is.

When I want to call the GP tool, I just need to pass in a class that specifies the parameters I need to pass in.
